src/UI/images/Image.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from '../../styles.scss';
+
+export default function Image({ image }) {
```

Comment:
add some prop types to the component please https://reactjs.org/docs/typechecking-with-proptypes.html